### PR TITLE
finish the hidden menu left

### PR DESCRIPTION
**Pull request by Ewar22**
Finish hidden menu

# Description
Function to hide menu

## Screenshots

image
image

### CODE REVIEW

**Review by DiegoMoralesBravo — APPROVED** [review]

File: app/javascript/navbarService.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  // document.getElementById("mainService").style.background = "none";
+  document.getElementById("logoService").style.display = "none";
+
+}
+
+const closeNav = () => {
+  document.getElementById("mySidenavService").style.width = "0";
+  document.getElementById("mainService").style.marginLeft= "0";
+  document.getElementById("logoService").style.display = "block";
+}
```
Comment (JorgeRodz) [inline app/javascript/navbarService.js:13]: Add an empty line at the end.

File: app/views/share/_menu.html.erb [hunk 2]
```diff
@@ -1,33 +1,20 @@
...
-
-<script type = "text/javascript">
-  $(document).ready(function(){
-  $('.sub-btn').click(function(){
-    $(this).next('.sub-menu').slideToggle();
 
-  });
-});
-</script>
+</div>
```
Comment (JorgeRodz) [inline app/views/share/_menu.html.erb:20]: Add an empty line at the end.

**Review by JorgeRodz — CHANGES_REQUESTED** [review]

**Review by claunicole — APPROVED** [review]
LGTM!

**Review by JorgeRodz — APPROVED** [review]
Nice work! :+1:

**Pull request merged**